documentation/external_libraries.rst

```diff
@@ -0,0 +1,105 @@
+External libraries
```

Comment:
Add a line above this for making linking easy in the future and also, since this is the top section add a line above it also, i.e., so we have ...

```rst
.. _external_libraries:

==================
External libraries
==================
...
```